.gitpod.yml

```diff
@@ -2,23 +2,31 @@ github:
   prebuilds:
     # enable for the default branch
     master: true
-# enable for all branches in this repo
+    # enable for all branches in this repo
     branches: false
-# enable for pull requests coming from this repo
+    # enable for pull requests coming from this repo
     pullRequests: true
-# enable for pull requests coming from forks
+    # enable for pull requests coming from forks
     pullRequestsFromForks: false
-# add a check to pull requests
+    # add a check to pull requests
     addCheck: true
-# add a "Review in Gitpod" button as a comment to pull requests
+    # add a "Review in Gitpod" button as a comment to pull requests
     addComment: false
-# add a "Review in Gitpod" button to the pull request's description
+    # add a "Review in Gitpod" button to the pull request's description
     addBadge: true
 
 image: gitpod/workspace-node-lts
 
+tasks:
+  - name: Install pnpm and dependencies
+    init: npm install -g pnpm@7.17.0
+    command: pnpm install
+  - name: Prepare the repositories
+    command: pnpm run prepare
```

Comment:
These two tasks should be merged, `pnpm install` should be in `init` and `pnpm run prepare` should be in `before`.